src/global/header.ts

```diff
@@ -125,8 +125,14 @@ async function createUserMenu (store: IndexedFormula, user: NamedNode): Promise<
 
   const throttledMenuToggle = throttle((event: Event) => toggleMenu(event, loggedInMenuTrigger, loggedInMenu), 50)
   loggedInMenuTrigger.addEventListener("click", throttledMenuToggle)
-  loggedInMenuContainer.addEventListener("mouseover", throttledMenuToggle)
-  loggedInMenuContainer.addEventListener("mouseout", throttledMenuToggle)
+  let timer = setTimeout(() => null, 0)
```

Comment:
Hmm, I'd prefer not allocating a noop onto the stack - is there a reason we're not leaving this undefined and checkif it it's set before calling `clearTimeout`?

Reply:
I thought it better to not introduce a test _every time_, but to make sure that there always is an instance that can be removed. Could replace it with simply number `0`, which I'm pretty certain would have the same effect.

Reply:
In that case I'd prefer it as-is, since browsers might choose to return `0` as the identifier for a different timeout already which we might not want to have cancelled. I don't feel strongly about this anyway, so I'll resolve this conversation :)